Terminal.Gui/ConsoleDrivers/NetDriver.cs

```diff
@@ -1310,25 +728,24 @@ public override void AddStr (ustring str)
 
 		public override void End ()
 		{
+			mainLoop.netEvents.StopTasks ();
+
 			if (IsWinPlatform) {
 				NetWinConsole.Cleanup ();
 			}
 
 			StopReportingMouseMoves ();
 			Console.ResetColor ();
-			Clear ();
+
+			//Disable alternative screen buffer.
+			Console.Out.Write ("\x1b[?1049l");
+			Console.Out.Flush ();
```

Comment:
Flush calls are not needed because AutoFlush is on, right?

Reply:
I really never set the `AutoFlush` and I always used the `Console.Out.Flush`. If it's true as default at least the use of the command never cause them to fail, may be if nothing exist to flush, the command ignore it. If you really want I will delete all theses commands.

Reply:
See https://learn.microsoft.com/en-us/dotnet/api/system.io.streamwriter.autoflush?view=net-7.0

<img width="524" alt="image" src="https://user-images.githubusercontent.com/585482/221299604-de173eec-1b50-48a9-89fa-bc8df78b32c9.png">

Reply:
Done. Removed all the `Console.Out.Flush ();` code.